Formula/netdata.rb

```diff
@@ -1,6 +1,6 @@
 class Netdata < Formula
-  desc "Distributed real-time performance and health monitoring"
-  homepage "https://my-netdata.io/"
+  desc "Instantly diagnose slowdowns and anomalies in your infrastructure with thousands of metrics, interactive visualizations, and insightful health alarms."
```

Comment:
```
==> brew style homebrew/core
==> FAILED
== Formula/netdata.rb ==
C:  2:119: Line is too long. [159/118]
C:  2:158: Description shouldn't end with a full stop
```

Can you make it short without `.` in the end?

Reply:
This needed a rebase before it was mergeable due to the version discrepancy (it was updated on `master`), so I took the liberty of doing this.